support/troubleshooting/troubleshooting-operating-system-issues.adoc

```diff
@@ -10,3 +10,42 @@ toc::[]
 
 // Investigating kernel crashes
 include::modules/investigating-kernel-crashes.adoc[leveloffset=+1]
+
+include::modules/troubleshooting-enabling-kdump.adoc[leveloffset=+2]
+
+include::modules/troubleshooting-enabling-kdump-day-one.adoc[leveloffset=+2]
+
+=== Testing the kdump configuration
+
+ifdef::openshift-enterprise[]
+See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/configuring-kdump-on-the-command-line_managing-monitoring-and-updating-the-kernel#testing-the-kdump-configuration_configuring-kdump-on-the-command-line[Testing the kdump configuration] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+ifdef::openshift-origin[]
+See the link:https://fedoraproject.org/wiki/How_to_use_kdump_to_debug_kernel_crashes#Step_2:_Capturing_the_Dump[Capturing the Dump] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+=== Analyzing a core dump
+
+ifdef::openshift-enterprise[]
+See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_monitoring_and_updating_the_kernel/analyzing-a-core-dump_managing-monitoring-and-updating-the-kernel[Analyzing a core dump] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+ifdef::openshift-origin[]
+See the link:https://fedoraproject.org/wiki/How_to_use_kdump_to_debug_kernel_crashes#Step_3:_Dump_Analysis[Dump Analysis] section in the {op-system-base} documentation for kdump.
+endif::[]
+
+[discrete]
+[role="_additional-resources"]
+=== Additional resources
```

Comment:
Should this be level 2?

Generally, we include an ID for al headers, such as

```
[id="{context}_additional_resources"]
```

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
I put it at level 3 to indicate that it's an additional resources section for "Investigating Kernel Crashes" only; I know it's the only content in this book, but it makes it easier if new content about something else is added later.

I'll add the ID, thanks!